configs/dsdl/README.md

```diff
@@ -0,0 +1,61 @@
+# DSDL Detection Dataset
+
```

Comment:
I think it should have been explained in one sentence what DSDL is at the beginning, instead of having to read a long paragraph to understand it. Can you add a sentence here to clarify?